schedule/application.go

```diff
@@ -24,8 +25,11 @@ func NewApplication(artisan console.Artisan, cache cache.Cache, log log.Log, deb
 	return &Application{
 		artisan: artisan,
 		cache:   cache,
-		log:     log,
-		debug:   debug,
+		cron: cron.New(cron.WithParser(cron.NewParser(
+			cron.SecondOptional|cron.Minute|cron.Hour|cron.Dom|cron.Month|cron.Dow|cron.Descriptor,
```

Comment:
The document: https://github.com/robfig/cron?tab=readme-ov-file#upgrading-to-v3-june-2019